src/lib/expansion/expansion-panel-header.ts

```diff
@@ -132,10 +141,14 @@ export class MatExpansionPanelHeader implements OnDestroy {
   }
 
   /** Gets whether the expand indicator should be shown. */
-  _showToggle(): boolean {
+  _showIndicator(): boolean {
     return !this.panel.hideToggle && !this.panel.disabled;
   }
 
+  _placeIndicatorBefore(): boolean {
```

Comment:
Does this function serve any value?

Reply:
This method determines the ordering of the content in the header.